Sources/Rendering/Core/ScalarBarActor/index.js

```diff
@@ -898,6 +922,10 @@ function vtkScalarBarActor(publicAPI, model) {
   publicAPI.resetAutoLayoutToDefault = () => {
     model.autoLayout = defaultAutoLayout(publicAPI, model);
   };
+
+  publicAPI.resetTicks = () => {
+    model.generateTicks = defaultGenerateTicks(publicAPI, model);
```

Comment:
you might want to call setGenerateTicks() instead of setting it manually (to make sure all the modifiedTime is updated)